Tests/SwiftSMTPTests/TestWrapUnwrap.swift

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Stephen Beitzel 2021
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+@testable import SwiftSMTP
+import XCTest
+
+class TestWrapUnwrap: XCTestCase {
+    var reallyLongLine: String = ""
+
+    override func setUpWithError() throws {
+        let address = "test_address@dumbster.local"
+        reallyLongLine = "To: "
+        for _ in 1...34 {
+            reallyLongLine.append(address)
+            reallyLongLine.append(", ")
+        }
+        reallyLongLine.append(address)
+    }
+
+    func testWrap() throws {
+        let wrapped = reallyLongLine.wrap(78)
+        let lines = wrapped.split(separator: "\r\n")
```

Comment:
For a more complete test, "\r\n" should probably have at least one of the whitespace characters after it.

Reply:
I'm not entirely sure what you're proposing to test with that change. However, just as a point of clarification, the argument `separator` in the `split` function must be a single character. Swift is interpreting the sequence "\r\n" and a single character, for whatever reason. It sure would be handy if String.split() could act like the split() function in Perl, for example.

Reply:
Actually, I've been thinking further about this and I think that the implementation I've proposed is really inadequate. I will be creating a few more unit tests to demonstrate the proper (RFC) behavior and then see about getting them to pass.